models/classes/runner/rubric/QtiRunnerRubric.php

```diff
@@ -51,6 +51,9 @@ public function getRubrics(RunnerServiceContext $context, $itemRef = null)
         if(!is_null($itemRef)){
             try {
                 $routeItem = $session->getRoute()->getRouteItemsByAssessmentItemRef($itemRef);
+                if ($routeItem) {
```

Comment:
May be you should check if `$routeItem` is an intanceof `qtism\runtime\tests\RouteItemCollection`  (or a parent class)